catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/AccessControlPolicyExtension.java

```diff
@@ -110,15 +123,22 @@ private KeyValueCollectionPermission isPermitted(
       return match; // Simply imply nothing early on (essentially a no-op in this extension)
     }
 
+    // To be able to have viewing access to the metacard, you must satisfy the following criteria
+    SecurityPredicate readPermsImplied =
+        (sub, mc) ->
+            hasAccessAdministrators.apply(sub, mc)
+                || hasAccessIndividuals.apply(sub, mc)
+                || hasAccessGroups.apply(sub, mc)
+                || hasAccessGroupsReadOnly.apply(sub, mc)
+                || hasAccessIndividualsReadOnly.apply(sub, mc);
+
     // get all permissions implied by the subject, this function returns what permissions
     // to filter from the key-value permission collection
     Supplier<Set<String>> impliedPermissions =
         () -> {
           if (isSystem.test(subject) || isOwner.apply(subject, metacard)) {
             return metacard.keySet(); // all permissions are implied
-          } else if (hasAccessAdministrators.apply(subject, metacard)
-              || hasAccessIndividuals.apply(subject, metacard)
-              || hasAccessGroups.apply(subject, metacard)) {
+          } else if (readPermsImplied.apply(subject, metacard)) {
```

Comment:
This looks like it is going to apply to read and write cases, not just read, see the above comment.